src/dashboard/get_chart_data/templates/get_chart_data.sql.jinja

```diff
@@ -0,0 +1,65 @@
+{%- import 'syntax.sql.jinja' as syntax -%}
+{%- import 'filter_numeric.sql.jinja' as numeric -%}
+{%-macro select_data(cumulus__none) -%}
+    SELECT 
+        {%- if stratifier_column %}
+        "{{ stratifier_column }}",
+        {%- endif %}
+        "{{ data_column }}"
+        {%- if count_columns %},{%- endif  %}
+        {%- for column in count_columns  %}
+        SUM("{{ column }}") AS {{ column }}{{ syntax.comma_delineate(loop) }}
+        {%- endfor%}
+    FROM "{{ schema }}"."{{ data_package_id }}"
+    WHERE
+        {%- if coalesce_columns  %} COALESCE (
+        {%- for column in coalesce_columns  %}
+            cast("{{ column }}" AS VARCHAR){{ syntax.comma_delineate(loop) }} 
+        {%- endfor%}
+        ) IS NOT NULL
+        AND{%- endif%} "{{ data_column }}" IS NOT NULL
+        {%- if not cumulus__none %}
+        AND {{ data_column }} !='cumulus__none'
+        {%- if filter_str %}
+        AND {{ filter_str }}
+        {%- endif  %}
+        {%- else %}
+        AND {{ data_column }} ='cumulus__none'
+        {%- endif  %}
+        {%- if stratifier_column %}
+        AND "{{ stratifier_column }}" IS NOT NULL
+        {%- endif  %}
+    GROUP BY
+        {%- if stratifier_column %}
+        "{{ stratifier_column }}", "{{ data_column }}"
+        {%- else %}
+        "{{ data_column }}"
+        {%- endif %}
+    ORDER BY
+        {%- if stratifier_column %}
+        "{{ stratifier_column }}", "{{ data_column }}"
+        {%- else %}
+        "{{ data_column }}"
+        {%- endif %}
+    {%-  endmacro -%}
+WITH non_null AS (
+    {{ select_data(False) }}
+),
+has_null AS (
+    {{ select_data(True) }}
+)
+SELECT * FROM non_null
+{%- if numeric.has_filter(filter_configs) == 'True' %}
+WHERE {{ numeric.get_filters(filter_configs) }}
+{%- endif %}
+UNION
+SELECT * from has_null
+{%- if numeric.has_filter(filter_configs) == 'True' %}
+WHERE {{ numeric.get_filters(filter_configs) }}
+{%- endif %}
```

Comment:
Am I missing something? This looks the same as the non-null block above. Why separate out the nulls and non-nulls into two selects?